src/nimblepkg/options.nim

```diff
@@ -130,6 +134,7 @@ Nimble Options:
   -v, --version                   Print version information.
   -y, --accept                    Accept all interactive prompts.
   -n, --reject                    Reject all interactive prompts.
+  -l, --local                     Run in project local dependency mode
```

Comment:
I would still vote for ``localdeps``, it's better for it to be explicit and you have a short flag too.

Reply:
Done